pkg/schedule/config/name.go

```diff
@@ -0,0 +1,30 @@
+// Copyright 2024 TiKV Project Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package config
+
+type CheckerSchedulerName string
```

Comment:
In a later PR, the scheduler will also use this type.

Reply:
Why using one type, not checkerName and schedulerName separately?

It is because I found they sometimes mix.
https://github.com/tikv/pd/blob/41e63768870fd86aaf78f499503a601249a7db39/pkg/schedule/filter/counter.go#L77-L93